custom_components/xiaomi_miio_airpurifier/fan.py

```diff
@@ -252,6 +257,30 @@
     ATTR_BUTTON_PRESSED: "button_pressed",
 }
 
+AVAILABLE_ATTRIBUTES_AIRPURIFIER_MIOT = {
+    # set fav rpm
```

Comment:
I'm not entirely sure what this means/why it's here. I'll check my notes later this week